### DISCUSSION TITLE
amp-ad not working in AMP page

### DISCUSSION THREAD

**Discussion by jatinvmehta** [vercel/next.js#13201] [Help]
We are under development for AMP page. We use hybrid option for our pages where we have separate components for AMP pages. We are trying to make amp-ad work as per documentation on amp.dev.

We are only seeing blank div and no ads get render at all. We are using Google AdSense and followed every bit of instruction.

Thanks in advance

**Comment by c0b41** [comment]
they eventually show up, just give 1-2 two days, and make sure wrapper div width and height.
